Linux Python Compile Failure

Failure to compile for Linux Python CI:
```
-- Found OpenCL: /usr/lib64/libOpenCL.so (found version "2.2") 
CMake Error at /usr/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
-- Configuring incomplete, errors occurred!
  Could NOT find Python (missing: Python_INCLUDE_DIRS Python_LIBRARIES
See also "/work/_skbuild/linux-x86_64-3.8/cmake-build/CMakeFiles/CMakeOutput.log".
  Development Development.Module Development.Embed) (found version "2.7.5")
See also "/work/_skbuild/linux-x86_64-3.8/cmake-build/CMakeFiles/CMakeError.log".
Call Stack (most recent call first):
  /usr/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.23/Modules/FindPython.cmake:561 (find_package_handle_standard_args)
  _skbuild/linux-x86_64-3.8/cmake-build/_deps/clci-src/CMakeLists.txt:16 (find_package)


  File "/opt/python/cp38-cp38/lib/python3.8/site-packages/skbuild/setuptools_wrap.py", line 544, in setup

    env = cmkr.configure(cmake_args,

  File "/opt/python/cp38-cp38/lib/python3.8/site-packages/skbuild/cmaker.py", line 221, in configure
    raise SKBuildError(
An error occurred while configuring with CMake.
  Command:
    "cmake" "/work" "-G" "Ninja" "-DCMAKE_INSTALL_PREFIX:PATH=/work/_skbuild/linux-x86_64-3.8/cmake-install" "-DPYTHON_EXECUTABLE:FILEPATH=/opt/python/cp38-cp38/bin/python" "-DPYTHON_VERSION_STRING:STRING=3.8.13" "-DPYTHON_INCLUDE_DIR:PATH=/opt/_internal/cpython-3.8.13/include/python3.8" "-DPYTHON_LIBRARY:FILEPATH=libpython3.8.a" "-DSKBUILD:BOOL=TRUE" "-DCMAKE_MODULE_PATH:PATH=/opt/python/cp38-cp38/lib/python3.8/site-packages/skbuild/resources/cmake" "-DITK_DIR:PATH=/work/ITK-cp38-cp38-manylinux_2_28_x64" "-DITK_USE_SYSTEM_SWIG:BOOL=ON" "-DWRAP_ITK_INSTALL_COMPONENT_IDENTIFIER:STRING=PythonWheel" "-DSWIG_EXECUTABLE:FILEPATH=/work/ITK-cp38-cp38-manylinux_2_28_x64/Wrapping/Generators/SwigInterface/swig/bin/swig" "-DCMAKE_CXX_COMPILER_TARGET:STRING=x86_64-linux-gnu" "-DBUILD_TESTING:BOOL=OFF" "-DPython3_EXECUTABLE:FILEPATH=/opt/python/cp38-cp38/bin/python" "-DPython3_INCLUDE_DIR:PATH=/opt/python/cp38-cp38/bin/../include/python3.8" "-DCMAKE_BUILD_TYPE:STRING=Release"
```

## Notes
- Python path input to CMake appear to match those in https://github.com/InsightSoftwareConsortium/ITKSplitComponents/runs/8162235914?check_suite_focus=true
- ITKSplitComponents CMake warns about Python paths, maybe our path hints are case-sensitive / do not exactly match what FindPython is trying to use?
```

2022-09-02T18:53:20.4583802Z -- Generating done
2022-09-02T18:53:20.4685154Z CMake Warning:
2022-09-02T18:53:20.4685758Z   Manually-specified variables were not used by the project:
2022-09-02T18:53:20.4685963Z 
2022-09-02T18:53:20.4686046Z     PYTHON_EXECUTABLE
2022-09-02T18:53:20.4686268Z     PYTHON_INCLUDE_DIR
2022-09-02T18:53:20.4686498Z     PYTHON_VERSION_STRING
2022-09-02T18:53:20.4686662Z 

```
- Check ITKPythonBuilds download to ensure version matches `itk-wheel-tag`

--- a/wrapping/dockcross-manylinux-build-module-wheels-opencl.sh
+++ b/wrapping/dockcross-manylinux-build-module-wheels-opencl.sh
@@ -8,8 +8,13 @@
 #
 #   scripts/dockcross-manylinux-build-module-wheels.sh cp39
 
+MANYLINUX_VERSION="_2_28"
+IMAGE_TAG=20220715-9ce3707
+OPENCL_ICD_LOADER_TAG=v2021.04.29
+OPENCL_HEADERS_TAG=v2021.04.29
+
 # Generate dockcross scripts
-docker run --rm dockcross/manylinux2014-x64:20211006-2a1c5fb > /tmp/dockcross-manylinux-x64
+docker run --rm dockcross/manylinux${MANYLINUX_VERSION}-x64:${IMAGE_TAG} > /tmp/dockcross-manylinux-x64
 chmod u+x /tmp/dockcross-manylinux-x64
 
 script_dir=$(cd $(dirname $0) || exit 1; pwd)
@@ -17,12 +22,12 @@ script_dir=$(cd $(dirname $0) || exit 1; pwd)
 if ! test -d ./OpenCL-ICD-Loader; then
   git clone https://github.com/KhronosGroup/OpenCL-ICD-Loader
   pushd OpenCL-ICD-Loader
-  git checkout v2021.04.29
+  git checkout ${OPENCL_ICD_LOADER_TAG}
   popd
   pushd OpenCL-ICD-Loader/inc
   git clone https://github.com/KhronosGroup/OpenCL-Headers
   pushd OpenCL-Headers
-  git checkout v2021.04.29
+  git checkout ${OPENCL_HEADERS_TAG}
   popd
   cp -r OpenCL-Headers/CL ./
   popd

--- a/wrapping/dockcross-manylinux-download-cache.sh
+++ b/wrapping/dockcross-manylinux-download-cache.sh
@@ -7,8 +7,9 @@ curl https://data.kitware.com/api/v1/file/592dd8068d777f16d01e1a92/download -o z
 gunzip -d zstd-1.2.0-linux.tar.gz
 tar xf zstd-1.2.0-linux.tar
 
-curl -L https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/download/${ITK_PACKAGE_VERSION:=v5.2.1}/ITKPythonBuilds-linux.tar.zst -O
+curl -L https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/download/${ITK_PACKAGE_VERSION:=v5.3rc04.post2}/ITKPythonBuilds-linux.tar.zst -O
 ./zstd-1.2.0-linux/bin/unzstd ITKPythonBuilds-linux.tar.zst -o ITKPythonBuilds-linux.tar
+echo "Extracting all files"
 tar xf ITKPythonBuilds-linux.tar
 
 mkdir tools

